docs/intro/reusable-apps.txt

```diff
@@ -262,6 +262,26 @@ this. For a small app like polls, this process isn't too difficult.
    you add some files to it. Many Django apps also provide their documentation
    online through sites like `readthedocs.org <https://readthedocs.org>`_.
 
+   Many Python projects, including Django and Python itself, use `Sphinx
+   <https://www.sphinx-doc.org/en/master/usage/quickstart.html>`_ to build
+   their documentation. If you choose to use Sphinx you can link back to the
+   Django documentation using `Intersphinx
+   <https://www.sphinx-doc.org/en/master/usage/quickstart.html#intersphinx>`_
+   by including a value for Django in your project's ``intersphinx_mapping``
+   configuration::
```

Comment:
```suggestion
   Django documentation by configuring `Intersphinx
   <https://www.sphinx-doc.org/en/master/usage/quickstart.html#intersphinx>`_
   and including a value for Django in your project's ``intersphinx_mapping``
   value::
```
I'm not sure if this is clearer but folks would need to follow the instructions in order to configure Intersphinx (they need to add `'sphinx.ext.intersphinx'` to the `extensions` list and the `intersphinx_mapping` mapping value is within the Sphinx `conf.py`/Sphinx configuration file).
So hope this implies there are more steps in the process and this is within the Sphinx conf

Reply:
Yes! Thanks. That's much better.